libs/core/src/components/pds-input/docs/pds-input.mdx

```diff
@@ -113,8 +113,82 @@ When `invalid` is true, the input displays the error message and provides a visu
+Prepend and append elements are placed outside the input container, next to the input field. These are typically used for larger elements or controls that need to be visually separated from the input.
 
-[MDN Web Docs: Input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input)
+<DocCanvas client:only
+  mdxSource={{
+    react: '<PdsInput componentId="pds-input-prepend-append-example" label="Amount" type="text" value="100"><pds-select class="pds-input__prepend" slot="prepend"><option value="USD">USD</option><option value="EUR">EUR</option></pds-select><pds-select class="pds-input__append" slot="append"><option value="1">1</option><option value="2">2</option></pds-select></PdsInput>',
+    webComponent: '<pds-input component-id="pds-input-prepend-append-example" label="Amount" type="text" value="100"><pds-select class="pds-input__prepend" slot="prepend"><option value="USD">USD</option><option value="EUR">EUR</option></pds-select><pds-select class="pds-input__append" slot="append"><option value="1">1</option><option value="2">2</option></pds-select></pds-input>'
+  }}>
+  <pds-input component-id="pds-input-prepend-append-example" label="Amount" type="text" value="100">
+    <pds-select class="pds-input__prepend" slot="prepend">
+      <option value="USD">USD</option>
+      <option value="EUR">EUR</option>
+    </pds-select>
+    <pds-select class="pds-input__append" slot="append">
+      <option value="1">1</option>
+      <option value="2">2</option>
+    </pds-select>
+  </pds-input>
+</DocCanvas>
+
+### Usage Guidelines
+
+- **Prefix/Suffix**: Use for small elements like icons or short text that should be visually integrated with the input field.
+- **Prepend/Append**: Use for larger elements like dropdowns or buttons that need to be visually separated from the input field.
+
+**Prefix Icon**
+<pds-input component-id="pds-input-prefix-suffix-example" label="Name" type="email" value="Frank Dux">
+  <pds-icon name="mail" slot="prefix"></pds-icon>
+</pds-input>
+
+**Prepend Select**
+<pds-input component-id="pds-input-prefix-suffix-example" label="Name" type="email" value="Frank Dux">
+  <pds-select class="pds-input__prepend" slot="prepend">
+    <option value="1">1</option>
+    <option value="2">2</option>
+    <option value="3">3</option>
+  </pds-select>
+</pds-input>
+
+**Prefix Icon and Append Select**
+<pds-input component-id="pds-input-prefix-suffix-example" label="Name" type="email" value="Frank Dux">
+  <pds-icon name="dollar" slot="prefix"></pds-icon>
+  <pds-select class="pds-input__append" slot="append">
+    <option value="1">USD</option>
+    <option value="2">EUR</option>
+    <option value="3">GBP</option>
+  </pds-select>
+</pds-input>
+
+**Unstyled button suffix**
+<pds-input component-id="pds-input-prefix-suffix-example-2" label="Name" type="email" value="Frank Dux">
+  <pds-button class="pds-input__suffix" variant="unstyled" slot="suffix"><pds-icon name="remove-circle"></pds-icon></pds-button>
```

Comment:
Would the author be expected to apply the `__suffix`/`__append`/`__prepend` classes, or is this something we could do by detecting the slot?